bin/fin

```diff
@@ -115,10 +115,29 @@ is_katacoda ()
 	[[ "$KATACODA" != "" ]]
 }
 
-is_github_actions () {
+is_github_actions ()
+{
 	[[ "$GITHUB_RUN_ID" != "" ]]
 }
 
+# Returns x86_64 / aarch64 (aka "old school")
+host_arch () {
+	case "$(uname -m)" in
+		x86_64) echo "x86_64" ;;
+		aarch64) echo "aarch64" ;;
+		* ) false;;
+	esac
+}
+
+# Returns amd64 / arm64 (aka "new style")
+host_arch_new () {
+	case "$(uname -m)" in
+		x86_64) echo "amd64" ;;
```

Comment:
@lmakarov should `x86_64` here be a different value?

Reply:
@lpeabody `uname -m` returns `x86_64` or `aarch64`.
Both `host_arch` and `host_arch_new` function use it but return different architecture styles.

- `x86_64` = `amd64`
- `aarch64` =  `arm64`

Reply:
Ah okay that clears up some confusion I had.